### ISSUE TITLE
fzf plugin not aware of debian package

### ISSUE THREAD

**Issue by gorsil** [robbyrussell/oh-my-zsh#7998]
I run Debian testing/sid on my machine and every new shell spits out: 
```
[oh-my-zsh] fzf plugin: Cannot find fzf installation directory.
 Please add `export FZF_BASE=/path/to/fzf/install/dir` to your .zshrc
```

Whilst I have fzf package installed:
```
dpkg -l | grep fzf

ii  fzf                               0.18.0-2                     amd64        general-purpose command-line fuzzy finder
```

Upon further investigation its pretty obvious that debain package installs fzf in a weird way that results in project files being all over the place:
<details><summary>Files for package</summary>

```
dpkg-query -L fzf

/usr
/usr/bin
/usr/bin/fzf
/usr/bin/fzf-tmux
/usr/share
/usr/share/bash-completion
/usr/share/bash-completion/completions
/usr/share/bash-completion/completions/fzf
/usr/share/doc
/usr/share/doc/fzf
/usr/share/doc/fzf/README.Debian
/usr/share/doc/fzf/changelog.Debian.gz
/usr/share/doc/fzf/changelog.gz
/usr/share/doc/fzf/copyright
/usr/share/doc/fzf/examples
/usr/share/doc/fzf/examples/fish_user_key_bindings.fish
/usr/share/doc/fzf/examples/fzf.vim
/usr/share/doc/fzf/examples/key-bindings.bash
/usr/share/doc/fzf/examples/key-bindings.fish
/usr/share/doc/fzf/examples/key-bindings.zsh
/usr/share/doc/fzf/examples/plugin
/usr/share/fish
/usr/share/fish/vendor_functions.d
/usr/share/fish/vendor_functions.d/fzf_key_bindings.fish
/usr/share/gocode
/usr/share/gocode/src
/usr/share/gocode/src/github.com
/usr/share/gocode/src/github.com/junegunn
/usr/share/gocode/src/github.com/junegunn/fzf
/usr/share/gocode/src/github.com/junegunn/fzf/go.mod
/usr/share/gocode/src/github.com/junegunn/fzf/go.sum
/usr/share/gocode/src/github.com/junegunn/fzf/main.go
/usr/share/gocode/src/github.com/junegunn/fzf/src
/usr/share/gocode/src/github.com/junegunn/fzf/src/algo
/usr/share/gocode/src/github.com/junegunn/fzf/src/algo/algo.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/algo/algo_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/algo/normalize.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/ansi.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/ansi_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/cache.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/cache_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/chunklist.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/chunklist_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/constants.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/core.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/history.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/history_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/item.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/item_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/matcher.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/merger.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/merger_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/options.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/options_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/pattern.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/pattern_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/reader.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/reader_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/result.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/result_others.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/result_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/result_x86.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/terminal.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/terminal_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/terminal_unix.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/terminal_windows.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/tokenizer.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/tokenizer_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/tui
/usr/share/gocode/src/github.com/junegunn/fzf/src/tui/dummy.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/tui/light.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/tui/tcell.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/tui/ttyname_unix.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/tui/ttyname_windows.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/tui/tui.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/tui/tui_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/util
/usr/share/gocode/src/github.com/junegunn/fzf/src/util/atomicbool.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/util/atomicbool_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/util/chars.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/util/chars_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/util/eventbox.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/util/eventbox_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/util/slab.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/util/util.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/util/util_test.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/util/util_unix.go
/usr/share/gocode/src/github.com/junegunn/fzf/src/util/util_windows.go
/usr/share/man
/usr/share/man/man1
/usr/share/man/man1/fzf-tmux.1.gz
/usr/share/man/man1/fzf.1.gz
/usr/share/zsh
/usr/share/zsh/vendor-completions
/usr/share/zsh/vendor-completions/_fzf
/usr/share/doc/fzf/examples/plugin/fzf.vim
```
</details>

According to these 2 pages the very same package is available for ubuntu:
https://packages.debian.org/sid/utils/fzf
https://qa.debian.org/developer.php?login=cdluminate%40gmail.com
I do not have means to confirm this theory, nevertheless there is very little chance that ubuntu and debian packages differ in any way.
ATM plugin works fine if fzf installed through pacman (archlinux only pretty much), brew or manual installation via git (https://github.com/junegunn/fzf#using-git)
It's pretty reasonable to expect plugin picking up system package if it's available.

@maxbrunet FYI 
I've already started working on fixing it, will submit PR soon

**Comment by maxbrunet** [comment]
Hi @gorsil, thank you for the ping. I can reproduce with:

```shell
IMAGE='debian:buster-slim'
docker run --rm -i "${IMAGE}" <<EOF
set -e
apt-get update
apt-get install --yes --no-install-recommends ca-certificates git zsh fzf
git clone https://github.com/robbyrussell/oh-my-zsh.git "\${HOME}/.oh-my-zsh"
sed 's/^plugins=.*$/plugins=(fzf)/' "\${HOME}/.oh-my-zsh/templates/zshrc.zsh-template" > "\${HOME}/.zshrc"
zsh -i -c exit
EOF

<omitted output>

Cloning into '/root/.oh-my-zsh'...
[oh-my-zsh] fzf plugin: Cannot find fzf installation directory.
 Please add `export FZF_BASE=/path/to/fzf/install/dir` to your .zshrc
```

Same for Ubuntu with `IMAGE='ubuntu:disco'`.

Please let me know when your PR is ready too, I'll give it a review!

**Comment by gorsil** [comment]
Hey @maxbrunet, everything is ready, check it out:
https://github.com/robbyrussell/oh-my-zsh/pull/8001

**Comment by mcornella** [comment]
Fixed in #8001.
